jerry-core/jrt/jrt.h

```diff
@@ -204,7 +204,7 @@ extern void __noreturn jerry_fatal (jerry_fatal_code_t);
  *
  * Returns minimum positive value, that divides @a alignment and is more than or equal to @a value
  */
-#define JERRY_ALIGNUP(value, alignment) ((alignment) * (((value) + (alignment) - 1) / (alignment)))
+#define JERRY_ALIGNUP(value, alignment) (((value) + ((alignment) - 1)) & ~((alignment) - 1))
```

Comment:
This works only if `alignment` is a power of two, while original version worked for any positive integer number. True enough, power-of-2 alignments are used in practice, but then please update the doc accordingly.